sdk/lib/rtl/sysvol.c

```diff
@@ -511,12 +513,15 @@ RtlpSysVolTakeOwnership(IN PUNICODE_STRING DirectoryPath,
 Cleanup:
     if (TokenEnabled)
     {
-        ZwAdjustPrivilegesToken(hToken,
-                                FALSE,
-                                &TokenPrivileges,
-                                0,
-                                NULL,
-                                NULL);
+        /* Disable privileges that we had to enable, whetever the result was. */
+        NTSTATUS Status2 = ZwAdjustPrivilegesToken(hToken,
+                                                   FALSE,
+                                                   &TokenPrivileges,
+                                                   0,
+                                                   NULL,
+                                                   NULL);
+        /* This must succeed */
+        ASSERT(NT_SUCCESS(Status2));
```

Comment:
Does `Status2` need `#if DBG` or the like?

Reply:
I'll `(void)` it, that will be simpler.